### DISCUSSION TITLE
Unit of the Output of Noise Simulation

### DISCUSSION THREAD

**Discussion by rupokdas147** [ra3xdh/qucs_s#1088] [Q&A] [answered]
I am performing noise simulations in Qucs-S and plotting the variable onoise_spectrum. I want to confirm the unit of onoise_spectrum. Does it represent the output noise spectral density in V^2/Hz (noise power) or in V/sqrt(Hz) (root spectral density)?
Also, if the default is V^2/Hz, do I need to manually take the square root to obtain V/sqrt(Hz)?
I would appreciate any clarification or examples from Qucs-S or Ngspice users!

![image.png](https://github.com/user-attachments/assets/f6e6ca6f-12c7-4c82-9407-9a4d9bb0f15f)

**Comment by dwarning** [comment, marked as answer]
From ngspice manual:
The units of all result vectors can be changed by using control variable sqrnoise:
• set sqrnoise: will deliver results in squared form, means the unit is V2/Hz or A2/Hz .
This value refers more to the convenient Power Spectral Density.
Default setting of ngspice is unset sqrnoise, which delivers Voltage or Current Noise Spectral Density. This is more practical from designers point of view.

**Reply by rupokdas147** [reply]
Thanks. Finally I found it on Ngspice manual.

**Comment by ra3xdh** [comment]
@rupokdas147 I have reopened this discussion. The discussions (not issues) must be kept open to make them visible for other users experiencing the similar problem. Just select one reply as answer instead.

**Reply by rupokdas147** [reply]
@ra3xdh Oh Sorry. Understood.
